.github/workflows/build-apps.yml

```
@@ -2,7 +2,12 @@ name: build-apps
 on:
   push:
     branches:
-      - v2 # @TODO: change branch to master after release of v2
+      - FEN-98-build-ios # @TODO: change branch to master
```

Will commit before merging
```suggestion
      - master
```